spec/helpers/moments_helper_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+describe MomentsHelper do
+  describe 'moments_stats' do
```

Comment:
It looks like the tests you moved into this file don't work, I highly recommend running this spec locally on your machine instead of relying on CircleCI to run it for you. From the root folder of the project simply run `rspec ./spec/helpers/moments_helper_spec.rb`

Reply:
It looks like user1 cannot be found. That's because user1 is undefined. If you take a look at the original application_controller_spec.rb you moved this test from you'll see user1 comes from `let(:user1) { create(:user1) }`